packages/components/date-picker/src/instance.ts

```diff
@@ -0,0 +1,12 @@
+import type { ComponentPublicInstance } from 'vue'
+import type { DatePickerProps } from './props/date-picker'
+
+export type DatePickerExpose = {
+  focus: (focusStartInput: boolean) => void
+  handleOpen: () => void
+  handleClose: () => void
+}
+export type DatePickerInstance = ComponentPublicInstance<
```

Comment:
@btea 

It's a different way from instances of other existing components, does it matter?

## Example

```
// /collapse/src/instance.ts
import type Collapse from './collapse.vue'
import type CollapseItem from './collapse-item.vue'

export type CollapseInstance = InstanceType<typeof Collapse>
export type CollapseItemInstance = InstanceType<typeof CollapseItem>
```